package.json

```diff
@@ -1,14 +1,14 @@
 {
   "name": "moxiedoc",
-  "version": "0.0.1",
+  "version": "0.0.2",
   "description": "Documentation tool",
   "author": "Johan Sörlin <spocke@moxiecode.com>",
   "bugs": {
-    "url": "http://github.com/spocke/moxiedoc/issues"
+    "url": "https://github.com/tinymce/moxiedoc/issues"
   },
   "contributors": [],
   "engines": {
-    "node": ">=0.8.0"
+    "node": ">=14.0.0"
```

Comment:
This is a breaking change and could cause issues. I'd suggest being a little more lenient here and making this `>=12.0.0` (the previous LTS version). Also given it's a breaking change, it should have a major version bump, however this is a `0.x` release so maybe we should just do a minor instead? eg 0.1.0

Reply:
I should also add I've not seen anything that actually requires a new version of NodeJS, so it doesn't really make sense to be changing this.

Reply:
Minor version bump and reduced nodejs version in https://github.com/tinymce/moxiedoc/pull/3/commits/0cb3456fcffb83f14aaca1998038cefacc650336

Reply:
Reversed this to 0.8.0 to avoid updating docker image at this time.